pkg/kubernetes/client.js

```diff
@@ -580,20 +581,65 @@ define([
                 });
         }
 
-        /* Load the kube config, and then try to start connecting */
-        cockpit.spawn(["/usr/bin/kubectl", "config", "view", "--output=json", "--raw"])
-            .fail(function(ex, output) {
-                if (output)
-                    console.warn(output);
-                else
-                    console.warn(ex);
-            })
-            .done(function(data) {
-                var scheme = config.parse_scheme(data);
+        if (token) {
```

Comment:
Well the oauth is really pluggable. I can easily imagine a case where someone replaces the login script with something that  does launch the real bridge *and* includes a kubeconfig file.